tests/test_scheduler.py

```diff
@@ -96,20 +96,15 @@ def tearDown(self):
 
     def test_add_experiment(self):
         app = scheduler.SchedulerApp(name="name", parent=QObject())
-        data = [ExperimentInfo(str(i), {"rid": i, "priority": 10 - i}) for i in range(10)]
-        for info in data:
-            app.addExperiment(info)
-        self.assertEqual(app.schedulerFrame.model.experimentQueue, data)
-        app.schedulerFrame.model.experimentQueue = []
-        permutation = [1, 9, 3, 8, 7, 4, 2, 6, 5, 0] # "priority" value for each experiment
-        inv_permutation = [1, 3, 4, 7, 8, 5, 2, 6, 0, 9] # experiment number when sorted
+        priorities = [1, 9, 3, 8, 7, 4, 2, 6, 5, 0] # "priority" value for each experiment
+        sorted_indices = [1, 3, 4, 7, 8, 5, 2, 6, 0, 9] # experiment number when sorted
```

Comment:
You can remove the comments now because the variable names are descriptive enough.

BTW we put 2 spaces before the comment "#".
See also: [PEP8](https://peps.python.org/pep-0008/#inline-comments)